packages/hdwallet-tallyho/src/tallyho.ts

```diff
@@ -222,6 +224,61 @@ export class TallyHoHDWallet implements core.HDWallet, core.ETHWallet {
     return false;
   }
 
+  private async detectTallyProvider(): Promise<TallyHoEthereumProvider | null> {
```

Comment:
Add the below comment?

```ts
  /*
   * Tally works the same way as metamask.
   * This code is copied from the @metamask/detect-provider package
   * @see https://www.npmjs.com/package/@metamask/detect-provider
   */
```

Reply:
Added in 2f22d667a2298e09b5215abb41de74343a6b0ce1